drakrun/drakrun/util.py

```diff
@@ -120,6 +121,42 @@ def get_xen_commandline(parsed_xl_info):
     return cfg
 
 
+def exception_handler(
```

Comment:
Sorry for being so snippy on this, but I think this adds unnecessary complexity to the code. Can you just rename the original `raise_if_true` to something more related to `create_rekall_profile` and we will be good to go:)

Reply:
Just curious, what did you expect when you said to make it more generic?

Reply:
The reason why I initialy suggested to rename `raise_if_true` function, or to make it more generic was actually the naming of the function. Since `raise_if_true` was used in only one context, it made sense to me to give it a meeningful name, e.g. `on_create_rekall_profile_failure`.

But if we wanted to keep the name, I guess that the function should be more generic then. What I mean by generic is a function that is not only parametrized with a lot of arguments, but can actually be used in different places in code. I admit that for now I don't have a clear vision on how that would look like. Thats why I suggested to stick with the rename fix.

Reply:
Ah okay, thanks for a detailed explanation. Will revert the changes :)